interop/deposit_contract_testnets/README.md

```diff
@@ -0,0 +1,69 @@
+# Guidelines for Eth2 testnets using a deposit contract
+
+This document provides a set of implementation guidelines for clients, aiming to simplify the interoperability testing in Eth2 phase 0 testnets using an Eth1 validator deposit contract.
+
+## Client-operated testnets
+
+An Eth2 testnet can be identified by an address of a deployed validator deposit contract, running on the Görli network. Multiple testnets can be running at the same time and all client teams are encouraged to deploy as many contracts as they see fit. 
+
+To facilitate discovering and connecting to existing testnets, all client teams are encouraged to publish metadata files for their active testnets in the following repository:
+
+https://github.com/eth2-clients/eth2-testnets
+
+Please create a sub-folder for your team, where each active testnet will be stored as a nested sub-folder with a descriptive name. For example: 
+
+```
+trinity/1000_validators_testnet/bootstrap_nodes.txt
+trinity/1000_validators_testnet/deposit_contract.txt
+trinity/1000_validators_testnet/config.yaml
+trinity/1000_validators_testnet/genesis.ssz
+```
+
+`bootstrap_nodes.txt` is a line-delimited text file with [multiaddr](https://github.com/multiformats/multiaddr) records for the bootstrap nodes of the testnet. For example:
+
+```
+/ip4/10.20.30.40/tcp/9100/p2p/16Uiu2HAmEAmp4FdpPzypKwTMmsbCdnUafDvXZCpFrUDbYJZNk7hX
+/ip4/10.20.30.50/tcp/9100/p2p/16Uiu2HAmV5SpcmrEymmaj1mymSHkbMpMVUXFRiXRxpHkfWdQ7bi3
+```
+
+To connect to the testnet, the user should either pass the entire file as a command-line parameter named `bootstrap_file` or she should specify the individual entries with a repeatable command-line parameter named `bootstrap_node`.
+
+`deposit_contract.txt` is a single-line text file with the address of the testnet's deposit contract on the Görli network. To connect to the testnet, the user should specify the address as a command-line parameter named `deposit_contract`.
+
+The `genesis.ssz` represents the genesis state snapshot. The user should specify the path to this file with a command-line parameter named `state_snapshot`.
+
+The `config.yaml` file is optional and it specifies the [Eth2 constants parameters](https://github.com/ethereum/eth2.0-specs/tree/dev/configs) used by the testnet. When no config file is present, it should be assumed that the testnet is using the `minimal` config. Users are expected to check each client's documentation for instructions regarding the usage of config preset files. Config presets are usually specified either at build-time or at run-time depending on the client.
+
+Any additional information and instructions for interacting with the specific testnet can be provided in a README file in the same folder.
+
+### Making deposits
+
+To obtain the 32 GöETH required for making a deposit, please use the social faucet at https://faucet.goerli.mudit.blog/
```

Comment:
Very minor, but perhaps we should use `MAX_EFFECTIVE_BALANCE` instead of `32`? Prysm are already using <32 deposits and it seems this is likely to happen again, especially if it's going to take about a month to get 32 ETH from the faucet (w/o Sybil attacking it).

Reply:
I'm happy to change this. My own thinking was that it would be quite easy to change the faucet since ChainSafe is controlling it (summoning @GregTheGreek to confirm) and the general public would have less reasons to be confused if we use the real deposit values in the testnets.

Reply:
So if we drop the total required eth to 512k we can run a full simulation on goërli (provided we have a refund option)

Reply:
Just out of interest, where does the 512k come from?

Reply:
@djrtwo has a proposal on the way

Reply:
I can get even more Goerli, just then i need to nudge @soc1c

Reply:
https://github.com/ethereum/eth2.0-specs/pull/1467

Reply:
for the deposit contract, do we want a drain function so we can retrieve these deposits back later? we don't have to do that if we have a lot of Goerli ETH of course

Reply:
That would be the most appreciated :)